05-select-from-a-dropdown/python/es5_dropdown.py

```diff
@@ -0,0 +1,39 @@
+# http://elementalselenium.com/tips/5-select-from-a-dropdown
+import unittest
+from selenium import webdriver
+# for test_example_2
+from selenium.webdriver.support.select import Select as WebDriverSelect
+
+
+class ES5_DropDown(unittest.TestCase):
+
+    def setUp(self):
+        self.driver = webdriver.Firefox()
+
+    def tearDown(self):
+        self.driver.quit()
+
+    def test_example_1(self):
+        driver = self.driver
+        driver.get('http://the-internet.herokuapp.com/dropdown')
+        dropdown_list = driver.find_element_by_id('dropdown')
+        options = dropdown_list.find_elements_by_tag_name('option')
+        for opt in options:
+            if opt.text == 'Option 1':
+                opt.click()
```

Comment:
If there is only one **Option 1** it would be a good idea to add `break` so that we don't waste time checking the other options. Otherwise, this is fine.